src/isolate.rs

```diff
@@ -102,6 +102,19 @@ impl Isolate {
         }
     }
 
+    /// Schedules an exception to be thrown when returning to JavaScript.
+    ///
+    /// When an exception has been scheduled it is illegal to invoke any
+    /// JavaScript operation; the caller must return immediately and only after
+    /// the exception has been handled does it become legal to invoke JavaScript
+    /// operations.
+    pub fn throw_exception(&self, exception: &::value::Value) -> ::value::Value {
```

Comment:
Interesting to import `Value` using an absolute path here, I don't mind so much but it doesn't match the rest of the code...